### ISSUE TITLE
Error in MS Edge ("Security Problem")

### ISSUE THREAD

**Issue by n1ghtmare** [sindresorhus/normalize-url#80]
First of all, sorry If this was "reported" or "asked" before, however I've been facing the following issue on MS Edge (note that Chrome, Firefox and Safari are all working just fine), and I'm unable to resolve it.

![normalize-url-bug-1](https://user-images.githubusercontent.com/3255810/49653648-08b15580-fa4f-11e8-9a16-74d49b0fcbe6.JPG)

More specifically it errors out on the following line:

![normalize-url-bug-2](https://user-images.githubusercontent.com/3255810/49653675-1f57ac80-fa4f-11e8-85be-32297c8a5ece.JPG)

Steps to reproduce:

run `create-react-app my-new-app-name`
then `npm install --save normalize-url`

Then simply try `normalizeUrl("www.google.com")` in the `App` component (or anywhere for that matter). Works fine everywhere _except_ MS Edge (v. 42.17134.1.0).

I apologize if this is something silly and obvious (like adding a polyfill of some sorts). Just for the record, after some desperate googling, I've tried adding `url-polyfill` and `babel-polyfill` and importing them in the app:

```
import "url-polyfill";
import "babel-polyfill";
```

Didn't really help. 

Any help is highly appreciated.

**Comment by n1ghtmare** [comment]
Ok, I _think_ I found the problem (I hope it's helpful in any way). It appears that MS Edge doesn't like the following:

https://github.com/sindresorhus/normalize-url/blob/e02a3aaa149ddf34ba4076adc0885c26718b7a37/index.js#L63-L66

If I set `stripAuthentication` to `true` (or if I remove `urlObj.username` & `urlObj.password` from the code) everything seems to work just fine.

It's something that MS Edge does when running `.toString()` on the `URL` when `username` and `password` are defined (even if empty).

It seems to be working fine on other browsers.
